client/app/scripts/superdesk-monitoring/monitoring.js

```diff
@@ -546,7 +546,6 @@
                                 list.style.height = (scope.total * ITEM_HEIGHT) + 'px';
                             }
 
-                            list.style.paddingTop = (from * ITEM_HEIGHT) + 'px';
```

Comment:
hey, I think it's needed.. if you have longer list and you scroll further down this should position the items properly. without having the scroll bar jumping.